ush/ufsda/yamltools.py

```diff
@@ -128,19 +168,32 @@ def calc_time_vars(config):
     bkg_time_obj = valid_time_obj
     config['BKG_YYYYmmddHHMMSS'] = bkg_time_obj.strftime('%Y%m%d.%H%M%S')
     config['BKG_ISOTIME'] = bkg_time_obj.strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
What is the ISOTIME?

Reply:
This distinguishes in the YAML `20221031.120000` (like for FV3) and the ISO8601 string `2022-10-31T12:00:00Z`

Reply:
Thanks Cory.